tests/Microsoft.Identity.Test.Integration.net45/SeleniumTests/SSHCertificateTests.cs

```diff
@@ -73,12 +77,20 @@ public async Task Interactive_SSHCert_Async()
             await MsalAssert.AssertSingleAccountAsync(labResponse, pca, result).ConfigureAwait(false);
         }
 
+
+        [TestMethod]
+        public async Task Interactive_SSHCert_MsaUser_Async()
```

Comment:
I don't think MSA can issue SSH certs, but it's a good test :)

Reply:
remvoed. thanks

---
In reply to: [348464362](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1511#discussion_r348464362) [](ancestors = 348464362)